examples/nextjs-lexical/package.json

```diff
@@ -5,7 +5,9 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
+    "dev:email": "email dev",
```

Comment:
did you mean to add this to the main nextjs-lexical package?

Reply:
Ah thanks I need to remove it 👍🏻